src/kudu/server/server_base.cc

```diff
@@ -358,9 +360,7 @@ shared_ptr<MemTracker> CreateMemTrackerForServer() {
 ServerBase::ServerBase(string name, const ServerBaseOptions& options,
                        const string& metric_namespace)
     : name_(std::move(name)),
-#ifdef FB_DO_NOT_REMOVE
       minidump_handler_(new MinidumpExceptionHandler()),
```

Comment:
do we really need the minidump handler?
This will need us to link with breakpad which turned out to be a problem.

Reply:
I thought it was needed for log deleter. I think I can get rid of it.